correlation_study.py

```diff
@@ -269,13 +305,10 @@ def generate_feature_dfs(stock1, stock2, window='1H', fillna_method=None):
             # Calculate rolling correlation
             rolling_corr = rolling_correlation(stock1, stock2, window)
 
-            # Create DataFrames for each stock
-            df_stock1 = self.dataframe[[stock1]].copy()
-            df_stock1['correlation'] = rolling_corr
-            
-            df_stock2 = self.dataframe[[stock2]].copy()
-            df_stock2['correlation'] = rolling_corr
-
+            # Create DataFrames for each stock, and attach the correlation df to stock2
+            df_stock1 = self.dataframe[[stock1.name]].copy()
+            df_stock2 = self.dataframe[[stock2.name]].copy()
+            df_stock2['Rolling_Correlation_Coefficient'] = rolling_corr
```

Comment:
Wouldn't it be better to have another output with the correlation coefficient directly? If one where to look for the Rolling_Correlation_Coefficient it's not intuitive to look for it in df_stock2. If you think it's better keeping it there please write it in the docstring